XiEditor/HoverViewController.swift

```diff
@@ -0,0 +1,120 @@
+// Copyright 2018 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Cocoa
+
+class HoverView: NSTextView {
+    // Lets AppKit do all the work when setting up a new text view
+    override init(frame frameRect: NSRect, textContainer container: NSTextContainer?) {
+        let textView = NSTextView(frame: .zero)
+        super.init(frame: frameRect, textContainer: textView.textContainer)
```

Comment:
this override doesn't seem to actually do anything. What happens if we just leave it out?